packages/ListBox/src/store/actionTypes.js

```diff
@@ -0,0 +1,24 @@
+export const accept = "ACCEPT";
+export const applyFilter = "APPLY_FILTER";
+export const cancel = "CANCEL";
+export const clear = "CLEAR";
+export const closePopover = "CLOSE_POPOVER";
+export const hideOptions = "HIDE_OPTION";
+export const openPopover = "OPEN_POPOVER";
+export const reset = "RESET";
+export const selectMultipleOption = "SELECT_MULTIPLE_OPTION";
```

Comment:
Was the choice to use camelCase instead of ALL_CAPS a deliberate one? I believe that in Redux it's typical to use ALL_CAPS for actions.

Reply:
I was expecting all caps, too.

Reply:
how big is your giveAShitMeter in this? is something that might be addressed in an incoming PR? this will not affect the ListBox is more an "author experience"

Reply:
I'm just interested in setting a convention before it propagates as a best practice to other components.  I'm okay with either choice, just don't want to have to change _more_ code later because we didn't give enough shits now.

Reply:
This is part of an answer that I post about the reducer pattern answering a question from alex. I have a list of things I want to improve for this component, but I don't think this is something that matter for V1. 

- Following best practices for reducer accordingly redux
- Allowed override of reducers
- And Alex suggest separate dispatch and state into two contexts

Reply:
this is the full list I have so far and is on my ux-status page. I can convert it into an issue if you feel more comfortable:

- [ ] Implementation  Advance Reducer Pattern
- [ ]<Trigger.Tag>
- [ ] Refactor reducer with Redux suggestions and implement override reducer as Kent dods approach for reducer
- [ ] tokens cleaning process
- [ ] Create diagram about Listbox structure
- [ ] Add two context one for dispatch one for the state

Reply:
This is a good list, I think adding it as an issue probably makes the most sense... but what is your "ux-status page"?